gridsome.config.js

```diff
@@ -85,19 +85,11 @@ module.exports = {
         ...vueRemarkCommonOptions
       }
     },
-      // Terms Nodes
-    {
-      use: '@gridsome/source-filesystem',
-      options: {
-        path: './content/terms/**/*.md',
-        typeName: 'Term'
-      }
-    },
-    require('./src/plugins/gridsome-plugin-th2-normalize-docs'),
```

Comment:
Why normalize-docs were renamed to terms?

Reply:
This plugin was created initially to manage versioned folders. Now it only connect pages and terms.